rotkehlchen/db/schema.py

```diff
@@ -490,8 +492,8 @@
 );
 """
 
-DB_CREATE_UNISWAP_EVENTS = """
-CREATE TABLE IF NOT EXISTS uniswap_events (
+DB_CREATE_AMM_EVENTS = """
```

Comment:
Don't do that. Make an issue to rename this in the next DB upgrade. Take these changes out of the PR.

As you can see, nothing else needs to change so there is no need to do a DB upgrade just for this. Use the old name **temporarily** and next time we do a DB upgrade, we also rename this. Make an issue to remind us.